### DISCUSSION TITLE
MAE of energy in the training output

### DISCUSSION THREAD

**Discussion by tgmaxson** [mir-group/nequip#251] [Q&A] [answered]
As the loss function is built on the MAE of energy when using "PerAtomMSELoss", it would make more sense to output the MAE per atom in the standard out while training.  Is there any way to output this via a setting or does this require some manual modification?

**Comment by nw13slx** [comment, marked as answer]
```
  - - total_energy
    - mae
    - PerAtom: True
```

Try this for the metric. See `full.yaml` for moredetails. And the loss function is by default on MSE, not MAE.
